modules/hocon/src/main/scala/ciris/hocon/package.scala

```diff
@@ -0,0 +1,33 @@
+package ciris
+
+import cats.Show
+import ciris.{ConfigException => _, _}
+import com.typesafe.{config => tsc}
+import cats.syntax.all
+
+import scala.util.Try
+
+package object hocon extends HoconConfigDecoders {
+
+  final class HoconAt(configV: ConfigValue[tsc.Config]) {
+
+    def apply(name: String): ConfigValue[tsc.ConfigValue] =
+      ConfigValue.suspend {
+        val key = ConfigKey(s"hocon key $name")
+        configV.flatMap { config =>
+          try {
+            val value = config.getValue(name)
```

Comment:
Can this throw anything other than `ConfigException.Missing`? Just wondering if the `ConfigValue.suspend` in `apply` is necessary.

Reply:
no, it cant, at least according to the docs. I dont really trust any java lib in this area but I will remove the suspend. But why its in the `env` example in the docs then? I would expect environment to be constant throughout the execution of the program.

Reply:
If we can find a single case where a different exception is thrown, then it warrants using `suspend`. I assume `Config` is immutable here? Or that would also warrant using `suspend`. In the case of `env`, it can throw `SecurityException` if the `SecurityManager` has been configured to disallow reading environment variables. You are right that environment variables are static though.